src/pybind/mgr/dashboard_v2/controllers/rbd.py

```diff
@@ -68,3 +84,29 @@ def get(self, pool_name):
         if status == ViewCache.VALUE_EXCEPTION:
             raise value
         return {'status': status, 'value': value}
+
+    def create(self, data):
+        if not self.rbd:
+            self.rbd = rbd.RBD()
+
+        # Get input values
+        size = data.get('size', 4 * 1024 ** 3)
```

Comment:
I'm not sure it makes sense to have a default size for RBD images on the backend: shouldn't the creator always have an idea of how big they want their image?

Reply:
This code is inspired by the related openATTIC RBD model: https://github.com/openattic/openattic/blob/master/backend/ceph/models.py#L806
There we defined a default value. But I agree, we can remove it.

Reply:
I'd also vote for making `size` mandatory here.

Reply:
Okay